pkg/insight/filestore.go

```diff
@@ -68,6 +72,36 @@ func (s *Store) PutChunk(ctx context.Context, chunk Chunk) error {
 	return s.filestore.PutObject(ctx, path, data)
 }
 
+func LoadChunksFromCache(cache cache.Cache, projectID, appID string, kind model.InsightMetricsKind, step model.InsightStep, from time.Time, count int) ([]Chunk, error) {
+	var chunks Chunks
+	paths := determineFilePaths(projectID, appID, kind, step, from, count)
```

Comment:
Because we know the size of chunks, so its capacity should be specified.

`chunks := make([]Chunk, 0, len(paths))`

Reply:
OK, thanks